### DISCUSSION TITLE
Docs for Github Action: How to install yq for a bash script?

### DISCUSSION THREAD

**Discussion by guettli** [mikefarah/yq#2232] [Q&A]
**Please describe your feature request.**

I want to use `yq` in a bash script.

I am looking at the current docs:

https://mikefarah.gitbook.io/yq/usage/github-action

I only see a way to run an action. I want to use `yq` in a bash script which I run in a Github Workflow step.

Please add instructions how to do that.

I tried apt and snap and both methods for installing did not work.

I found that: https://github.com/frenck/action-setup-yq

But somehow I would like to avoid it. It is one more dependency, and one more person who could inject evil code.

(and of course: Thank you very much for yq. I use it almost daily)

**Comment by mikefarah** [comment]
If you don't want to use the github action I provide, I'd either use the docker image or wget/curl from the release page I guess.

Note that my github action just extends from the official dockerfile: `https://github.com/mikefarah/yq/blob/master/github-action/Dockerfile`
